File: app/src/main/java/com/vikho305/isaho220/outstanding/data/Comment.java
package com.vikho305.isaho220.outstanding.data;

import com.google.gson.annotations.SerializedName;

public class Comment {
    private String id;
    private String text;
    @SerializedName(value = "date_created")
    private String dateCreated;
    private Post post;
    private User author;

    public String getId() {
        return id;
    }

    public String getText() {
        return text;
    }

    public String getDateCreated() {
        return dateCreated;
    }

    public Post getPost() {
        return post;
    }

    public User getAuthor() {
        return author;
    }
}

File: app/src/main/java/com/vikho305/isaho220/outstanding/data/Comment.kt
package com.vikho305.isaho220.outstanding.data

import com.google.gson.annotations.SerializedName

class Comment {
    val id: String? = null
    val text: String? = null

    @SerializedName(value = "date_created")
    val dateCreated: String? = null
    val post: Post? = null
    val author: User? = null
}